### PR TITLE
Update React transform READMEs from babel.github.io [skip ci]

### PR DESCRIPTION
**Pull request by existentialism**
For babel/babel.github.io#990 and babel/babel.github.io#1002, small batch for easy review.

Will follow up with website PR.

### CODE REVIEW

**Comment by hzoo** [comment]
You also mean https://github.com/babel/babel.github.io/issues/997 right?

**Comment by existentialism** [comment]
@hzoo yeah lol oops

File: packages/babel-plugin-transform-react-inline-elements/README.md [hunk 1]
```diff
@@ -1,11 +1,58 @@
 # babel-plugin-transform-react-inline-elements
 
-Turn JSX elements into exploded React objects
+Replaces the `React.createElement` function with one that is more optimized for production: `babelHelpers.jsx`.
+
+<blockquote class="babel-callout babel-callout-info">
```
Comment (hzoo) [inline packages/babel-plugin-transform-react-inline-elements/README.md]: Awesome stuff!

I think the only thing is to keep this <blockquote> in the website readme and then import the rest of the readme from here.

Comment (existentialism) [inline packages/babel-plugin-transform-react-inline-elements/README.md]: :+1:

File: packages/babel-plugin-transform-react-constant-elements/README.md [hunk 2]
```diff
@@ -1,11 +1,54 @@
 # babel-plugin-transform-react-constant-elements
 
-Treat React JSX elements as value types and hoist them to the highest scope
+Hoists element creation to the top level for subtrees that are fully static, which reduces both allocations and calls to `React.createElement`. More importantly, it tells React that the subtree hasn't changed so React can completely skip it when reconciling.
```
Comment (xtuc) [inline packages/babel-plugin-transform-react-constant-elements/README.md]: This paragraph will be cut out once on the website.

To be consistent, you are missing the description in comment (`>`). [for example](https://github.com/babel/babel/blob/master/packages/babel-preset-es2017/README.md).

**Pull request merged**